packages/editor/src/client/routes/imageList.tsx

```diff
@@ -98,7 +87,7 @@ export function ImageList() {
     }
   }, [current])
 
-  function loadMore() {
+  /* function loadMore() {
```

Comment:
Is there a reason that you uncommented loadMore?

Reply:
Yes. I'll need it soon for pagination which is not ready yet.